samples/Network/hdinsight-require-subscription-vnet/azurepolicy.json

```diff
@@ -0,0 +1,56 @@
+{
+    "mode": "all",
+    "policyRule": {
+      "if": {
+        "anyOf": [
+          {
+            "allOf": [
+              {
+                "field": "type",
+                "equals": "Microsoft.HDInsight/clusters"
+              },
+              {
+                "not": {
+                  "field": "Microsoft.HDInsight/clusters/computeProfile.roles[*].virtualNetworkProfile.subnet",
+                  "notIn": [
+                    "null",
```

Comment:
>null [](start = 21, length = 4)

Have you seen "null" (as a string) in your testing? 

This is a new one to me.

Reply:
This particular code snippet came from Chris, I havent actually seen a case myself, but am following it as best practice.